### PR TITLE
Update key expiration time, change key ids from 32bit to 64bit

### PR DESCRIPTION
**Pull request by darkk**
Closes TheTorProject/ooni-web#128

### CODE REVIEW

File: client/ngapp/views/about.html [hunk 1]
```diff
@@ -23,10 +23,10 @@
     <p>
     To contact the OONI team send an email to <a href="mailto:contact [AT] ooni.io">contact [AT] ooni.io</a>.
```
Comment (hellais) [inline client/ngapp/views/about.html]: While we are at it we should also change `contact@ooni.io` to `contact@openobservatory.org`

Comment (darkk) [inline client/ngapp/views/about.html]: I've also marked `openobservatory.org` as primary UID for `6B2943F00CB177B7` (`ooni.io` was primary one). But I'm unsure if it's properly uploaded to keyservers.

**Comment by hellais** [comment]
LGTM.
@anadahz can you re-deploy ooni-explorer with current master tip?

**Comment by anadahz** [comment]
https://github.com/TheTorProject/ooni-sysadmin/pull/103

**Pull request merged**